integrationtest/mapper/pojo/src/test/java/org/hibernate/search/integrationtest/mapper/pojo/mapping/definition/IndexNullAsValueIT.java

```diff
@@ -0,0 +1,227 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.pojo.mapping.definition;
+
+import java.lang.invoke.MethodHandles;
+import java.time.LocalDate;
+
+import org.hibernate.search.integrationtest.mapper.pojo.testsupport.util.rule.JavaBeanMappingSetupHelper;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.DocumentId;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.GenericField;
+import org.hibernate.search.mapper.pojo.mapping.definition.annotation.Indexed;
+import org.hibernate.search.util.common.SearchException;
+import org.hibernate.search.util.impl.integrationtest.common.rule.BackendMock;
+import org.hibernate.search.util.impl.test.SubTest;
+
+import org.junit.Rule;
+import org.junit.Test;
+
+public class IndexNullAsValueIT {
```

Comment:
There are many types that should be supported but are not, mainly the `java.time` types.

I know it's more verbose, but since this is a type-sensitive feature, I think it would be safer to rely on the `PropertyTypeDescriptors` to test it.

I think you should be able to just extend `org.hibernate.search.integrationtest.mapper.pojo.mapping.definition.FieldDefaultBridgeIT`. Simply add a new property in the mapped types (there is one for each `DefaultValueBridgeExpectations` implementation). Name the property `myPropertyWithIndexNullAs` for example, and add a `@GenericField` annotation with the appropriate "indexNullAs". You'll then be able to add a method in `FieldDefaultBridgeIT` that tests the nominal case.

To test for error cases, I think you'll be better off with a separate test. Let's talk about it once you get there?

Reply:
Good idea. Let's do that.